gcm/gcm.py

```diff
@@ -13,7 +13,7 @@
     from urllib.parse import unquote
 
 
-GCM_URL = 'https://gcm-http.googleapis.com/gcm/send'
+GCM_URL = 'https://fcm.googleapis.com/fcm/send/'
```

Comment:
You don't need the trailing slash.  It won't work with the slash

Reply:
Can you please test?

Reply:
Sorry it wasn't tested because I replaced the library in my stack. I was just trying to be helpful to the maintainer. I updated the pull request to remove the trailing slash, but doesn't look like it will be merged any time soon regardless.